crypto/crypto.ts

```diff
@@ -220,6 +220,17 @@ const stdCrypto: StdCrypto = ((x) => x)({
       data: BufferSource | AsyncIterable<BufferSource> | Iterable<BufferSource>,
     ): Promise<ArrayBuffer> {
       const { name, length } = normalizeAlgorithm(algorithm);
+
+      if (
+        length !== undefined &&
+        (length < 0 || length > maximumDigestLength ||
+          !Number.isInteger(length))
+      ) {
+        throw new RangeError(
+          `length must be an integer between 0 and ${maximumDigestLength}, inclusive`,
+        );
+      }
+
```

Comment:
I'm a bit inclined to go with the names `assertValidDigestLength` and `MAX_DIGEST_LENGTH`, because I find the alternatives somewhat imprecise.

`assertIs32BitInteger` would make me think that it's either asserting the range of a 32-bit unsigned integer, or a 32-bit signed integer, but in fact it's enforcing the upper bound of a 32-bit signed integer but the lower bound of zero/that of a 32-bit unsigned integer, and also allowing it to be undefined (if we entirely extract this logic). So a more accurate name would be like `assertIsNonNegative32BitSignedIntOrUndefined`.

I'm happy to switch to the names you suggested if you still prefer them, but just wanted to share my reasoning.

Reply:
Ok, I see. Yep, your suggested name makes sense 👍🏾